wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,77 @@
+# Installing and updating drivers 
+In this post, you will learn how to download or update drivers on your computer.
+It is highly recommended to avoid using third-party software to update or install your drivers.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly identify the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager. You can access the device manager by right-clicking the Windows start button or by pressing <kbd>Win</kbd> + <kbd>X</kbd> and selecting Device Manager. In the Device Manager, look for the device you require drivers for, and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you require drivers for.
+
+## Method 1: Using Windows Update
+You can usually find drivers for your drivers in Windows Update, instead of having to manually download the file.
```

Comment:
"drivers for your drivers" lol
```suggestion
You can usually find drivers for your devices in Windows Update, instead of having to manually download the file.
```